### ISSUE TITLE
 Why use the center of  axis-aligned box as the ground truth center in sunrgbd dataset?

### ISSUE THREAD

**Issue by Na-Z** [facebookresearch/votenet#29]
When you prepare training data for SUNRGBD dataset, the center of the axis-aligned box is treated as the ground truth center, according to the line 178-190, 198 in sunrgbd_detection_dataset.py.

My question is why use center of axis-aligned box rather than the center of the ground truth bbox (refer to "bbox3d_centers"  in the script)?

**Comment by dashidhy** [comment]
The axis-aligned box has the same center as the ground truth box has.
